src/StoreKeeper/WooCommerce/B2C/Core.php

```diff
@@ -146,7 +146,7 @@ public static function plugin_abspath()
     public function __construct()
     {
         add_action('init', function() {
-            load_plugin_textdomain('free-shipping', false, 'storekeeper-for-woocommerce/languages/');
+            load_plugin_textdomain(I18N::DOMAIN, false, 'storekeeper-for-woocommerce/i18n/');
```

Comment:
we already load it https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/blob/fc7841263dd0fc4a8081d9dcd555f93e5e4c7a88/src/StoreKeeper/WooCommerce/B2C/Core.php#L158